client/scripts/publish.js

```diff
@@ -1,9 +1,28 @@
-const { execSync } = require("child_process")
+const { section, linebreak, info, question, cmd } = require("./pretty-logs.js")
 
 const arg = process.argv[2]
 
-if (arg) {
-  execSync(`pnpm pack-module -- ${arg}`, { stdio: "inherit" })
-  process.chdir(`modules/${arg}/dist`)
-  execSync(`npm publish --access public`, { stdio: "inherit" })
-}
+;(async () => {
+  if (arg) {
+    cmd(`pnpm pack-module -- ${arg}`)
+
+    section("PUBLISH")
+    linebreak()
+
+    const answer = await question("Do you want to publish this package? [y/n] -> ")
```

Comment:
Change to `y/N` to show no is default.

Reply:
Will get fixed in the next PR (this file is getting deleted already lmao). Separate script will publish all packages at once because if the versioning/linking between packages gets messed up it will cause chaos